.github/workflows/test.yaml

```diff
@@ -6,24 +6,20 @@ on:
     paths:
```

Comment:
isn't that what `synchronize` is doing? the default types are `opened`, `synchronize`, and `reopened`, so this adds the test run when a label is applied (which if I haven't gotten myself too turned around within the actions is needed for when there are secrets being used and we don't want it to run without a maintainer adding a label).

Reply:
> this adds the test run when a label is applied 
exactly. we can remove `types` from test.yaml. but i've kept it in `netlifypreview.yml`, that way you don't have to push another commit after adding the label to get the website preview.

Reply:
And since we're taking out the labeling condition for `test.yml`, this becomes a moot point. I missed the connection yesterday - thanks for explaining!